CHANGES.rst

```diff
@@ -3,21 +3,27 @@
 
 New Tools and Services
 ----------------------
+gaia
+^^^^
 
+- TAP notifications service is now available for Gaia. If there is notification for the users,
+e.g planned or unplanned downtimes of the Gaia Archive, etc. [#2376]
 
 hsa
 ^^^
 
 - New module to access ESA Herschel mission. [#2122]
 
+
+
 Service fixes and enhancements
 ------------------------------
 
-casda
-^^^^^
-
-- Add the ability to produce 2D and 3D cutouts from ASKAP images and cubes. [#2366]
```

Comment:
Restored! Thanks

Reply:
The statement about not removing change log entries applies to all of them, not just this one entry individually.

Reply:
The statement about not removing change log entries applies to all of them, not just this one entry individually.

Reply:
@eerovaher - This is a rebase issue, I can fix it when the PR otherwise ready to be merged rather than nitpick on the need of multiple round of rebases.